packages/block-library/src/cover/edit.js

```diff
@@ -164,18 +199,12 @@ class CoverEdit extends Component {
 						<>
 							<MediaUploadCheck>
 								<Toolbar>
-									<MediaUpload
-										onSelect={ onSelectMedia }
-										allowedTypes={ ALLOWED_MEDIA_TYPES }
-										value={ id }
-										render={ ( { open } ) => (
-											<IconButton
-												className="components-toolbar__control"
-												label={ __( 'Edit media' ) }
-												icon="edit"
-												onClick={ open }
-											/>
-										) }
+									<IconButton
+										className={ classnames( 'components-icon-button components-toolbar__control', { 'is-active': this.state.isEditing } ) }
```

Comment:
The classname `components-icon-button` can be removed (it's already added by the `IconButton` component.)